books/free-programming-books-de.md

```diff
@@ -149,6 +150,11 @@
 * [LaTeX : Referenz der Umgebungen, Makros, Längen und Zähler](http://www.lehmanns.de/page/latexreferenz)
 
 
+### Mathematik
+
+* [Calcul mathématique avec SAGE](http://sagebook.gforge.inria.fr/english.html) - Paul Zimmermann et al.
```

Comment:
are you sure this is in german?

Reply:
The page is in English, but at the bottom there is a link to the German version.

> Traductions/Translations
>
> * German translation, contributed by Helmut Büch
> * French version (May 2013, 468 pages, for Sage 5.9)

Unfortunately there isn't a dedicated German page that hosts the download unlike with English and French.

Since for this it's just a PDF anyway, we could make the German one reference the link directly to skip the English/French page of that's preferred?

```md
[Title](http://www.loria.fr/~zimmerma/sagebook/CalculDeutsch.pdf) - Author (PDF)
```